lib/cors-anywhere.js

```diff
@@ -234,6 +237,7 @@ function getHandler(options, proxy) {
     requireHeader: null,            // Require a header to be set?
     removeHeaders: [],              // Strip these request headers.
     setHeaders: {},                 // Set these request headers.
+    maxAge: null,                   // If set, an Access-Control-Max-Age header with this value (in seconds) will be added.
```

Comment:
Default to 0 please, based on the following substeps from https://fetch.spec.whatwg.org/#cors-preflight-fetch

> 10. Let _max-age_ be the result of extracting header list values given \``Access-Control-Max-Age`\` and response’s header list.
> 11. If _max-age_ is failure or null, then set _max-age_ to zero.